spec/lib/declare_schema/model/foreign_key_definition_spec.rb

```diff
@@ -25,7 +25,7 @@ class Network < ActiveRecord::Base
     subject { described_class.new(model, foreign_key, options)}
 
     before do
-      allow(connection).to receive(:index_name).with('models', column: 'network_id') { 'on_network_id' }
+      allow(model.connection).to receive(:index_name).with(model.table_name, column: options[:foreign_key]&.to_s || foreign_key.to_s) { 'index_on_network_id' }
```

Comment:
Is this `with` important for the test? If not, you could just use `with(any_args)` .

Reply:
Great point!